docs_src/examples/Ch-ExamplesAddingModbusDevice.md

```diff
@@ -276,33 +435,143 @@ Now we're ready to run some commands.
 ### Find Executable Commands
 
 Use the following query to find executable commands:
+```
+$ curl http://your-edgex-server-ip:48082/api/v1/device | json_pp
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100  1718  100  1718    0     0  14081      0 --:--:-- --:--:-- --:--:-- 14081
+[
+   {
+      "id" : "56dcf3ad-52d8-4d12-a2d0-ae53c177ae3d",
+      "commands" : [
+         {
+            "put" : {
+               "url" : "http://edgex-core-command:48082/api/v1/device/56dcf3ad-52d8-4d12-a2d0-ae53c177ae3d/command/67b35f63-8f94-427b-a60c-188bf9e0633a",
+               "parameterNames" : [
+                  "DemandWindowSize",
+                  "LineFrequency"
+               ],
+               "path" : "/api/v1/device/{deviceId}/Configuration"
+            },
+            "id" : "67b35f63-8f94-427b-a60c-188bf9e0633a",
+            "get" : {
+               "url" : "http://edgex-core-command:48082/api/v1/device/56dcf3ad-52d8-4d12-a2d0-ae53c177ae3d/command/67b35f63-8f94-427b-a60c-188bf9e0633a",
+               "responses" : [
+                  {
+                     "description" : "service unavailable",
+                     "code" : "503"
+                  }
+               ],
+               "path" : "/api/v1/device/{deviceId}/Configuration"
+            },
+            ...
+            "name" : "Configuration"
+         }
+      ],
+      ...
+   },
+   {
+      ....
+   }
+]
+```
 
-    photon-ip:48082/api/v1/device
-
-| 
+### Execute PUT command
 
-![Executable Commands](commands.png)
+Execute PUT command according to `url` and `parameterNames`, replacing [host] with the server IP when running the edgex-core-command. This can be done in either of the following ways:
+
+```
+$ curl http://your-edgex-server-ip:48082/api/v1/device/56dcf3ad-52d8-4d12-a2d0-ae53c177ae3d/command/67b35f63-8f94-427b-a60c-188bf9e0633a \
+    -H "Content-Type:application/json" -X PUT  \
+    -d '{"DemandWindowSize":"1122","LineFrequency":"1012"}'
+```
+or
```

Comment:
Aside from using device id and command id in the URL, use the following API with device name and command is another approach.
Refer to [Core Command API] (https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-command/1.2.0#/default/put_v1_device_name__name__command__commandname_) for more details.

Reply:
Modified.

Reply:
The hyperlink doesn't show as my expectation, please modify it